### PR TITLE
Remove polyfills

### PR DESCRIPTION
**Pull request by dhoward**
Addresses #112 

So far this handles the low-hanging fruit (4 out of the 7 polyfills).

All explicit uses of `Set`, `Array.from`, and `Symbol` have been removed from the code as well. However, these are also used by Babel and removing them would require getting rid of a lot of ES6 features, such as the spread operator.

### CODE REVIEW

**Comment by dhoward** [comment]
Build failures should go away once #116 is merged

**Comment by doctyper** [comment]
@dhoward @cwelch5 Can we live with using `Object.assign({}, foo)` over `{...foo}` in Helmet?

**Comment by dhoward** [comment]
@doctyper then wouldn't we just need another polyfill for `Object.assign`? Pretty sure it isn't supported in IE

**Comment by doctyper** [comment]
@dhoward yeah we could use [`objectAssign`](https://github.com/sindresorhus/object-assign) or something similar. But at that point Helmet wouldn't depend on Symbols or any of the mentioned APIs.

**Comment by dhoward** [comment]
@doctyper ok ya that looks good

TIL the word ponyfill

**Comment by dhoward** [comment]
@cwelch5 @doctyper all polyfills are now gone!

The known ES6 features to avoid are destructuring, spread operators, and `for...of` iteration

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,3 @@
-import "core-js/fn/array/from";
-import "core-js/fn/array/includes";
-import "core-js/fn/map";
-import "core-js/fn/object/get-own-property-symbols";
-import "core-js/fn/object/is";
-import "core-js/fn/object/keys";
-import "core-js/fn/set";
 import Helmet from "./Helmet";
```
Comment (cwelch5) [inline src/index.js]: @doctyper @dhoward you guys think we should remove the index.js file as well, and change package.json to point directly to Helmet.js?  Would match Wildcat convention as well :-) just sayin'

Comment (doctyper) [inline src/index.js]: 👍

Comment (dhoward) [inline src/index.js]: @cwelch5 @doctyper index.js has been removed, and package.json points to Helmet.js. Also changed the import in the tests. Is that everything that needs to be changed?

Comment (cwelch5) [inline src/index.js]: I think you are good.

**Pull request merged**